server/utils/nunjucksSetup.ts

```diff
@@ -385,6 +392,45 @@ export default function nunjucksSetup(app: express.Application): nunjucks.Enviro
     }
   })
 
+  njkEnv.addFilter('toPrisonTypeCheckboxes', (allPrisonsFilter: AllPrisonsFilter) => {
+    return {
+      idPrefix: 'prisonTypeCode',
+      name: 'prisonTypeCodes',
+      classes: 'govuk-checkboxes--small',
+      fieldset: {
+        legend: {
+          text: 'Prison Types',
+          classes: 'govuk-fieldset__legend--m',
+        },
+      },
+      hint: {
+        text: 'Display selected prison types only',
+      },
+      items: [
```

Comment:
Thought about creating an endpoint in `prison-register` which returned a list of prison types, but thought that wasn't necessary as right now there isn't a requirement to introduce new ones. Perhaps in the future it will be needed